Neos.Neos/Classes/Command/NodeCommandControllerPlugin.php

```diff
@@ -166,6 +173,34 @@ public function invokeSubCommand($controllerCommandName, ConsoleOutput $output,
         }
     }
 
+    /**
+     * Creates the /sites node if it is missing.
+     *
+     * @param string $workspaceName Name of the workspace to consider (unused)
+     * @param boolean $dryRun Simulate?
+     * @return void
+     */
+    protected function createMissingSitesNode($workspaceName, $dryRun)
+    {
+        $this->output->outputLine('Checking for "%s" node ...', array(SiteService::SITES_ROOT_PATH));
+        $rootNode = $this->contextFactory->create()->getRootNode();
+        // We fetch the workspace to be sure it's known to the persistence manager and persist all
+        // so the workspace and site node are persisted before we import any nodes to it.
+        $rootNode->getContext()->getWorkspace();
+        $this->persistenceManager->persistAll();
+        $sitesNode = $rootNode->getNode(SiteService::SITES_ROOT_PATH);
+        if ($sitesNode === null) {
+            if ($dryRun === false) {
+                $rootNode->createNode(NodePaths::getNodeNameFromPath(SiteService::SITES_ROOT_PATH));
+                $this->output->outputLine('Created "%s" node', [SiteService::SITES_ROOT_PATH]);
+            } else {
+                $this->output->outputLine('Found missing "%s" node', [SiteService::SITES_ROOT_PATH]);
```

Comment:
This wording is a bit confusing.. Checking for "/sites" node ..., Found missing "/sites" node?